packages/space-opera/src/components/materials_panel/materials_panel.ts

```diff
@@ -120,8 +132,33 @@ export class MaterialPanel extends ConnectedLitElement {
     }
   }
 
-  getMaterial() {
-    return getModelViewer()!.model!.materials[this.selectedMaterialIndex];
+  getMaterial(options: GetMaterialOptions = GetMaterialOptions.Default) {
+    let material =
+        getModelViewer()!.model!.materials[this.selectedMaterialIndex];
+
+    // Creates a new instance if allowed.
+    if (options === GetMaterialOptions.AllowInstancing) {
```

Comment:
Maybe this should just be its own  function, instead  of using options to change the behavior?

Reply:
Done.